api/models.py

```diff
@@ -100,6 +88,14 @@ class Schedule(models.Model):
     def __str__(self):
         return self.time
 
+    @property
+    def running_time(self):
+        schedule = self.schedule
```

Comment:
음.. 이건 제가 말씀드린 걸 잘못 이해하신 것 같아요ㅠ
어떤 모델에서 메소드를 구현할 때에 필수 인자로 들어가는 self는 해당 모델의 인스턴스를 가리켜요
여기서의 self는 Schedule 모델의 한 인스턴스를 가리키게 되겠죠. 그래서 코드에서 의도한 대로라면 self.schedule이 아니라  그냥 그대로 self를 사용하시면 될것 같아요
```suggestion
        finish = datetime.strptime(self.finish_time, '%Y-%m-%dT%H:%M:%S%z')
        start = datetime.strptime(self.start_time, '%Y-%m-%dT%H:%M:%S%z')
```